open_alchemy/build/__init__.py

```diff
@@ -244,4 +244,13 @@ def execute(*, spec: typing.Any, name: str, path: str) -> None:
     init_models_file = generate_init_models_file(schemas=schemas)
     init = generate_init(open_alchemy=init_open_alchemy, models_file=init_models_file)
 
-    dump(path=path, name=name, setup=setup, manifest=manifest, spec=spec_str, init=init)
+    build_path = pathlib.Path(path) / name
```

Comment:
Do you think it would be better to move the decision about which folder to store the package in to the `dump` function? It is designed to do everything it needs to for output, already has the `name` parameter and has logic to deal with files (such as looking for folders and creating them if necessary).

Reply:
Oh you're right. That makes complete sense to update the path closer to the logic. I will change that.